graphql/query_test.go

```diff
@@ -44,9 +44,13 @@ func populateMainlineCommits(t *testing.T) {
 		versionId := fmt.Sprintf("v%d", i)
 		// Every other version is activated
 		isActivated := i%2 == 0
+		requester := evergreen.RepotrackerVersionRequester
```

Comment:
You may also want the occasional "trigger_request" https://evergreen.mongodb.com/version/evergreen_01c89d8b647ce46eff789e89c25eec83bc26fe8b_45c242d5e398741eeba433a0f34526e5 not sure if the new UI also has a way to optionally show that requester type